src/imitation/algorithms/adversarial/gail.py

```diff
@@ -3,16 +3,65 @@
 from typing import Optional
 
 import torch as th
-from stable_baselines3.common import base_class, vec_env
+from stable_baselines3.common import base_class
+from stable_baselines3.common.vec_env.base_vec_env import VecEnv
 from torch.nn import functional as F
 
 from imitation.algorithms import base
 from imitation.algorithms.adversarial import common
 from imitation.rewards import reward_nets
 
 
-class LogSigmoidRewardNet(reward_nets.RewardNet):
-    """Wrapper for reward network that takes log sigmoid of wrapped network."""
+class RewardNetFromDiscriminatorLogit(reward_nets.RewardNet):
+    r"""Converts the discriminator logits raw value to a reward signal.
```

Comment:
thanks for writing this explanation, the math checks out and is much clearer than before :)

Reply:
Glad to hear! :)